### PR TITLE
Refactor exception handling when loading the bindings

### PR DESCRIPTION
**Pull request by vxgmichel**
Now, all failing operations should end up raising a RuntimeError.

### CODE REVIEW

File: src/_cffi_src/build_bindings.py [hunk 1]
```diff
@@ -1,19 +1,14 @@
...
 
-# see: https://docs.python.org/3/library/platform.html#platform.architecture
-is_64bits = sys.maxsize > 2 ** 32
-
-
 BASEDIR = os.path.dirname(os.path.abspath(__file__))
 
 # import `get_winfsp_dir` the violent way given winfspy cannot be loaded yet
-get_winfsp_dir = None
 exec(open(f"{BASEDIR}/../winfspy/plumbing/get_winfsp_dir.py").read())
```
Comment (touilleMan) [inline src/_cffi_src/build_bindings.py]: We should be able to replace this by:
```python
g = {}
exec(open(f"{BASEDIR}/../winfspy/plumbing/get_winfsp_dir.py").read(), g) 
WINFSP_DIR = g['get_winfsp_dir']()
```

Hence `get_winfsp_dir` wouldn't become a function out of thin air (and require the `# noqa# tag)

File: src/winfspy/plumbing/get_winfsp_dir.py [hunk 2]
```diff
@@ -47,4 +47,19 @@ def get_winfsp_bin_dir():
     - the `bin` directory in the base directory provided by `get_winfsp_dir``
     """
     path = os.environ.get("WINFSP_DEBUG_PATH")
-    return path if path else get_winfsp_dir(suffix="bin")
+    if path:
+        path = pathlib.Path(path)
```
Comment (touilleMan) [inline src/winfspy/plumbing/get_winfsp_dir.py:51]: So now `get_winfsp_bin_dir` and `get_winfsp_dir` return `Path`, are we sure there is no place where it still assumes it returns str ?

**Review by touilleMan — APPROVED** [review]

File: src/_cffi_src/build_bindings.py (hunk 1 above)
Comment (vxgmichel) [reply src/_cffi_src/build_bindings.py]: :+1:

File: src/winfspy/plumbing/get_winfsp_dir.py (hunk 2 above)
Comment (vxgmichel) [reply src/winfspy/plumbing/get_winfsp_dir.py:51]: I'll double check!

**Pull request merged**